paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/CatalogTableITCase.java

```diff
@@ -132,6 +132,40 @@ public void testSnapshotsTable() throws Exception {
                         Row.of(3L, 0L, "APPEND"));
     }
 
+    @Test
```

Comment:
I think add a class of BinlogTableTest in package "org.apache.paimon.table.system" may better?